### ISSUE TITLE
Does "trice u" require TRICE macro on a single line?

### ISSUE THREAD

**Issue by escherstair** [rokath/trice#280]
I've noticed that if my C source code is written in this way
```
TRICE("the values are a: %d b: %d",
            3,
            5);
```
with the two parameters on separated lines,
`trice u` doesn't insert the Id(xxxx) inside the macro.
You can use numerica vlaues or variables, the behavior is the same.

If the code is on the same line
`TRICE("the values are a: %d b: %d", 3, 5);`
everything works fine.

So the question is: does `trice u` requires that `TRICE` macro is written on a single line?

**Comment by rokath** [comment]
You are right! I thought to have documented it, but probably forgot. Now done in the [TriceUserGuide](https://github.com/rokath/trice/blob/master/docs/TriceUserGuide.md#LimitationtriceurequiresTRICEmacrosonasingleline).
